test/support/govuk_content_schema_examples.rb

```diff
@@ -41,6 +41,19 @@ def stub_example_content_items_in_content_store
 
   module ClassMethods
     def govuk_content_schema_examples
+      supported_formats = %w{
```

Comment:
Probably should be `supported_examples` - not all the files here refer to distinct formats.